.github/workflows/test.yml

```diff
@@ -78,27 +78,52 @@ jobs:
             - name: Install dependencies
               run: yarn --frozen-lockfile
 
-            - name: Generate docs URL
-              id: docsURL
-              uses: actions/github-script@v6
-              with:
-                  script: |
-                      const { buildPreviewURLComment } = await import('${{ github.workspace }}/tasks/build-preview-urls-comment.js');
-                      const body = buildPreviewURLComment(process.env.GITHUB_HEAD_REF);
-                      const urlRegex = /(https?:\/\/[^\s]+)/g;
-
-                      const docsURL = body.match(urlRegex)?.[0];
-                      core.setOutput('docsURL', docsURL);
+            - name: Extract branch name
+              shell: bash
+              run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+              id: extract_branch
 
             - name: Lighthouse CI Action
               id: lighthouse
               uses: treosh/lighthouse-ci-action@v11
               with:
-                  urls: |
-                      ${{ steps.docsURL.outputs.docsURL }}
                   configPath: '.github/lighthouse/lighthouserc.json'
-                  uploadArtifacts: true
+                  runs: 3
                   temporaryPublicStorage: true
+                  uploadArtifacts: true
+                  urls: |
+                      https://opensource.adobe.com/spectrum-web-components/
+                      https://main--spectrum-web-components.netlify.app/
+                      https://${{ steps.extract_branch.outputs.branch }}--spectrum-web-components.netlify.app/
+
+            - name: Post Results
+              uses: actions/github-script@v6
+              with:
+                  script: |
+                      const { buildLighthouseComment } = await import('${{ github.workspace }}/tasks/build-lighthouse-comment.js');
+                      const body = buildLighthouseComment(${{ steps.lighthouse.outputs.links }}, ${{ steps.lighthouse.outputs.manifest }});
+                      github.rest.issues.listComments({
```

Comment:
I'm not sure what context this runs in... would `const data = await github.rest.issues.listComments(...)` work?

Reply:
It's no longer recent, but with the post initial creation move to `await import()` above, we should have access to that here and be able to flatten out the asynchrony here. I'll put this and the abstraction process into an issue to track and clarify steps. Great call!